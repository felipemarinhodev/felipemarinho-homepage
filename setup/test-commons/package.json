{
  "name": "@felipemarinho/test-commons",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "files": [
    "base.ts"
  ],
  "scripts": {
    "lint": "echo 'Not necessary'",
    "test": "echo 'Not necessary'"
  },
  "devDependencies": {
    "jest": "27.0.0",
    "jest-styled-components": "7.0.8",
    "ts-node": "10.7.0",
    "babel-jest": "28.0.3",
    "@babel/core": "7.17.10",
    "@babel/preset-env": "7.17.10",
    "@babel/preset-react": "7.16.7",
    "@babel/preset-typescript": "7.16.7",
    "@testing-library/react": "13.2.0"
  }
}
