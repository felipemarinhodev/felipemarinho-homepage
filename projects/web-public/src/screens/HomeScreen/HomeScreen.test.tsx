import { customRender } from "@felipemarinho/test-commons/react-testing-library";
import { HomeScreen } from ".";
const render = customRender();

describe("<HomeScreen />", () => {
  it("should render the component correctly", () => {
    const { container } = render(<HomeScreen />);
    expect(container).toMatchSnapshot();
  });
});
