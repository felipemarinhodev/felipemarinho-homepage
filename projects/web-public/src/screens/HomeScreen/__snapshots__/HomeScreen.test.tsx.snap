// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> should render the component correctly 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <main>
    <h1
      class="c0"
    >
      Home
    </h1>
    <p
      class="c0"
    >
      Importando modulo local: @felipemarinho/utils/math/sum sum(2,2) = 4
    </p>
    <p>
      Já demos diversas dicas envolvendo o aplicativo Calculadora (Calculator) do macOS aqui no MacMagazine. Na prática, elas acabam facilitando — e muito — a vida de quem faz muitos cálculos durante o dia. Na dica de hoje, mostraremos como você pode escolher quantas casas decimais serão exibidas nos resultados dos cálculos.
    </p>
  </main>
</div>
`;
